spec/features/manage_cmr/subscriptions/create_subscriptions_spec.rb

```diff
@@ -28,19 +28,21 @@
         end
 
         it 'displays validation errors within the form' do
-          expect(page).to have_content('Description is required.')
+          expect(page).to have_content('Name is required.')
```

Comment:
This should be 'Subscription Name', right?

Reply:
Yeah, I was going to make a few minor changes in this test file to align Name with 'Subscription Name' when I had other feedback.